vllm/config.py

```diff
@@ -1063,6 +1073,12 @@ class ParallelConfig:
 
     rank: int = 0
 
+    def compute_hash(self):
+        factors: List[Any] = []
```

Comment:
`disable_custom_all_reduce`?

And what about `worker_cls`?

Reply:
all reduce is wrapped as a custom op, and `disable_custom_all_reduce` only affects the internal of the custom op, it does not affect the fx graph.

worker class does not affect fx graph either. the model determines the fx graph.